content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,819 @@
+---
+title: "Migrating to the AWS SDK for Go V2"
+linkTitle: "Migrating"
+description: "How to migrate to the AWS SDK for Go V2 from AWS SDK for Go."
+---
+
+## Minimum Go Version
+
+The {{% alias sdk-go %}} requires a minimum version of Go 1.15. Migration from AWS SDK for Go to {{% alias sdk-go %}}
+may require you to upgrade your application by one or more Go versions. The latest version of Go can be downloaded on
+the [Downloads](https://golang.org/dl/) page. See the [Release History](https://golang.org/doc/devel/release.html) for
+more information about each Go version release, and relevant information required for upgrading.
+
+## Modularization
+
+The {{% alias sdk-go %}} has been updated to take advantage of the Go modules which became the default development mode
+in Go 1.13. A number of packages provided by the SDK have been modularized and are independently versioned and
+released respectively. This change allows for improved application dependency modeling, and will allow the SDK to
+to provide new features and functionality in a manner that follows the Go module versioning strategy.
+
+The following is are just some of the Go modules provided by the SDK:
```

Comment:
"The following is are just some of*" Maybe this could be simplified to "The following are some Go modules provided by the SDK:"